homeassistant/components/assist_satellite/entity.py

```diff
@@ -309,6 +329,114 @@ async def async_start_conversation(
         """Start a conversation from the satellite."""
         raise NotImplementedError
 
+    async def async_internal_ask_question(
+        self,
+        question: str | None = None,
+        question_media_id: str | None = None,
+        preannounce: bool = True,
+        preannounce_media_id: str = PREANNOUNCE_URL,
+        answers: list[dict[str, Any]] | None = None,
+    ) -> AssistSatelliteAnswer | None:
+        """Ask a question and get a user's response from the satellite.
+
+        If question_media_id is not provided, question is synthesized to audio
+        with the selected pipeline.
+
+        If question_media_id is provided, it is played directly. It is possible
+        to omit the message and the satellite will not show any text.
+
+        If preannounce is True, a sound is played before the start message or media.
+        If preannounce_media_id is provided, it overrides the default sound.
+
+        Calls async_ask_question.
+        """
+        await self._cancel_running_pipeline()
+
+        if question is None:
+            question = ""
+
+        if answers is None:
+            answers = []
```

Comment:
We shouldn't do this. Instead, we should skip calling `_question_response_to_answer` is `answers is None`